NEWS.md

```diff
@@ -10,13 +10,14 @@ vcr 0.5.4
 
 * changed `write_disk_path` handling internally to not run it through `normalizePath` before recording it to the cassette; passing the path through `normalizePath` was leading to the full path recorded in the cassette, which means in a package testing context that a test that uses a file on disk will (likely) only work on the machine the cassette was first created on. with relative paths in a package context, a test that has a file written on disk should now work in different testing contexts (locally, and various continuous integration platforms) (#135) (#166)
 * added a bit of documentation about large files created when using vcr, and how to ignore them if needed within `.Rinstignore` and/or `.Rbuildignore` (#164)
+* changed the name `helper-pkgname` to `setup-pkgname` to comply with `testthat`'s latest recommendations (#183, @maelle).
```

Comment:
this is the current version on cran - i create news items right before i submit to cran

Reply:
oops, ok, I removed the item.
do you keep track of changes anywhere or do you just use the commit history?

Reply:
use the issues and use `git difftool` to look at changes since last git tag (which is the last CRAN version)